spec/policies/cart_policy_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CartPolicy, type: :policy do
```

Comment:
after removing unnecessary policies from Cart CartItemds and other Classes you must clear many tests.

Reply:
skip